### ISSUE TITLE
webrtc push to talk

### ISSUE THREAD

**Issue by robrotheram** [RocketChat/Rocket.Chat#2058]
The voice chat is a nice feature but what it really needs is a push to talk setting that is bound to a user configured key press. Most people dont have good mics so echo can be annoying dedicated chat applications such as mumble have this feature.  It would also make this a big alternative to [discordapp.com](discordapp.com)

**Comment by skylord123** [comment]
Only thing holding me back at this point.

**Comment by elricsfate** [comment]
This would be a terrific self hosted replacement for something like Discord if this was implemented.

**Comment by Nemra1** [comment]
hope to  see this addons soon

**Comment by HammyHavoc** [comment]
Yeah this is something stopping me moving some contacts away from Discord.

**Comment by Tofdu31** [comment]
Hello everyone,

I wanted to know if the topic of Push to talk was advanced?

**Comment by Tofdu31** [comment]
After my request, which was a year ago, has the push to talk project progressed?

**Comment by VirtualAirwaves** [comment]
FYI @Tofdu31 -- WebRTC doesn't really support Push-To-Talk! We've been about to work around it with Virtual Airwaves (see it live at https://cb.virtualairwaves.com/ or learn more at https://virtualairwaves.com/ ) but we're going to see if we can get it in the spec so it's supported properly.

**Comment by Tofdu31** [comment]
@RobertSwirsky thank, but i think it's possible to create a Push To Talk server with WebRTC. Why not ?
